src/kakarot/instructions/system_operations.cairo

```diff
@@ -766,6 +709,38 @@ namespace CreateHelper {
         return (create2_address,);
     }
 
+    func get_evm_address{
```

Comment:
should this be in CreateHelper as well?

Reply:
not sure to understand. It **is** in CreateHelper, it's to avoid the if/else branch in the main flow and copy/paste of subsequent code